addons/calendar/models/calendar_event.py

```diff
@@ -569,6 +569,9 @@ def write(self, values):
         update_recurrence = recurrence_update_setting in ('all_events', 'future_events') and len(self) == 1
         break_recurrence = values.get('recurrency') is False
 
+        if any(vals in self._get_recurrent_fields() for vals in values) and not (update_recurrence or values.get('recurrency')):
+            raise UserError(_('Unable to Save the Recurrence with "This Event"'))
```

Comment:
Normally we don't use capitalized words in the middle of the sentence
```suggestion
            raise UserError(_('Unable to save the recurrence with "This Event"'))
```